src/xSTIR/mSTIR/tests/test1.m

```diff
@@ -106,6 +106,14 @@
 test.check(s)
 test.check(v, delta)
 
+% Check that the storage scheme for a pre-initialised object
+% doesn't change just because the default has changed.
+temp = PET.AcquisitionData();
+temp.set_storage_scheme('memory');
```

Comment:
I'd be amazed if this changes the default. In C++,Python you need `PET.AcquisitionData.set_storage_scheme`